src/core/main.cpp

```diff
@@ -99,6 +99,16 @@ void signalHandler( int signum ) {
 }
 #endif
 
+void interruptHandler(int code) {
+	using namespace lmms::gui;
+
+	if(getGUI() != nullptr) {
+		getGUI()->sendInterrupt(code);
+	} else {
+		exit(3);
+	}
+}
+
```

Comment:
I considered removing this and binding `signal(SIGINT` directly to the GuiApplication function, but I wanted to also capture the interrupt from CLI and I thought this was the cleanest way to handle both.